assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -1,15 +1,14 @@
+import {select} from 'd3-selection';
```

Comment:
This module has been refactored into a bunch of separate functions that render parts of the control. Where different parts need to interact, I get the appropriate component by selection rather than saving a bunch of variables. Also there is no linking to the Redux store. This component merely updates the state. We might have to revisit that if time is updated in a different component but I believe this simplifies things and prevents lots of rerendering when the state changes.